autogluon/training/buildspec_0_6_1.yml

```diff
@@ -0,0 +1,51 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Replace the file name from `_` to `-`. Also, rename the same for `buildspec_0_5_2.yml` file